gitpractice/javascript/task13.js

```diff
@@ -0,0 +1,25 @@
+/*http://www.codewars.com/kata/array-helpers*/
+Array.prototype.sum = function() { return this.reduce((sum, curr) => sum += curr, 0) }
+Array.prototype.square = function() { return this.map(x => x * x) }
+Array.prototype.cube = function() { return this.map(x => x * x * x) }
+Array.prototype.average = function() { return (this.reduce((sum, curr) => sum += curr, 0) / this.length) }
+Array.prototype.even = function() {
+    array = [];
+    console.log(this[0])
+    this.forEach((x) => { if (x % 2 == 0) array.push(x) });
+    //  (let i = 0; i < this.length; i++) {
+    //     if (this[i] % 2 == 0)
+    //         array.push(this[i])
+    // }
+    return array;
+}
+Array.prototype.odd = function() {
+    array = [];
+    console.log(this[0])
+    this.forEach(x => { if (x % 2 != 0) array.push(x) });
```

Comment:
you can use filter method, when you need to filter some data from the given set.

Use more array methods, if they are available (filter,map,etc)